experiments/conductor/cmd/runner/runner.go

```diff
@@ -390,6 +413,11 @@ func printHelp() {
 	log.Println("\t22 - [CRD] Generate CRD for each branch")
 	log.Println("\t23 - [CRD] Generate Mapper for each branch")
 	log.Println("\t24 - [CRD] Generate Fuzzer for each branch")
+	log.Println("\t40 - [Controller] Generate controller client for each branch")
+	log.Println("\t41 - [Controller] Generate controller for each branch")
+	log.Println("\t43 - [Controller] Create identity and reference files for each branch")
```

Comment:
This is done in `Generate Types for each branch" already.

Reply:
ill check if this generates better version. And if so move it to 21 or 22

Reply:
adding a comment in the help that this cmd (43) is optional.